c10/util/string_view.h

```diff
@@ -592,7 +592,10 @@ constexpr inline void swap(
     basic_string_view<CharT>& rhs) noexcept {
   lhs.swap(rhs);
 }
+
+#if !defined(C10_NODEPRECATED) || defined(USE_XPU)
```

Comment:
Does that hint at the fact that the XPU part of the codebase still uses it?
@guangy10 could we fix that? That should be a pretty simple codemod!